File: 프로그래머스/lv1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    let answer = [];
    for(let i = 1; i < n + 1; i++){
        if(n%i == 0 && m%i == 0) answer[0] = i;
    }
    answer[1] = n*m/answer[0];
    return answer;
}

File: 프로그래머스/lv1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
class Solution {
    public int[] solution(int n, int m) {
        int[] answer = new int[2];
        int bowl = 0;
        for(int i = 1; i < n + 1; i++){
            if(n%i == 0 && m%i == 0){
                answer[0] = i;
            }
        }
        answer[1] = n*m/answer[0];
        return answer;
    }
}